src/main/java/teammates/storage/entity/FeedbackResponseComment.java

```diff
@@ -72,26 +80,28 @@ private FeedbackResponseComment() {
         // required by Objectify
     }
 
-    public FeedbackResponseComment(String courseId, String feedbackSessionName,
-            String feedbackQuestionId, String giverEmail, String feedbackResponseId,
-            Instant createdAt, Text commentText,
-            String giverSection, String receiverSection, List<FeedbackParticipantType> showCommentTo,
-            List<FeedbackParticipantType> showGiverNameTo, String lastEditorEmail, Instant lastEditedAt) {
+    public FeedbackResponseComment(String courseId, String feedbackSessionName, String feedbackQuestionId,
+            String giverEmail, FeedbackParticipantType commentGiverType, String feedbackResponseId, Instant createdAt,
+            Text commentText, String giverSection, String receiverSection, List<FeedbackParticipantType> showCommentTo,
+            List<FeedbackParticipantType> showGiverNameTo, String lastEditorEmail, Instant lastEditedAt,
+            boolean isCommentFromFeedbackParticipant, boolean isVisibilityFollowingFeedbackQuestion) {
         this.feedbackResponseCommentId = null; // Auto generated by GAE
         this.courseId = courseId;
         this.feedbackSessionName = feedbackSessionName;
         this.feedbackQuestionId = feedbackQuestionId;
         this.giverEmail = giverEmail;
+        this.commentGiverType = commentGiverType;
         this.feedbackResponseId = feedbackResponseId;
         this.createdAt = TimeHelper.convertInstantToDate(createdAt);
         this.commentText = SanitizationHelper.sanitizeForRichText(commentText);
         this.giverSection = giverSection;
         this.receiverSection = receiverSection;
         this.showCommentTo = showCommentTo == null ? new ArrayList<FeedbackParticipantType>() : showCommentTo;
         this.showGiverNameTo = showGiverNameTo == null ? new ArrayList<FeedbackParticipantType>() : showGiverNameTo;
-        this.isVisibilityFollowingFeedbackQuestion = false;
```

Comment:
Because `isVisibilityFollowingFeedbackQuestion` used to be false for every instructor comment.